safaribooks.py

```diff
@@ -806,10 +806,10 @@ def _thread_download_images(self, url):
                                               stream=True)
             if response == 0:
                 self.display.error("Error trying to retrieve this image: %s\n    From: %s" % (image_name, url))
-
-            with open(image_path, 'wb') as img:
-                for chunk in response.iter_content(1024):
-                    img.write(chunk)
+            else:
```

Comment:
Resolves
```
[#] No connection adapters were found for 'mailto:shwetaimages@gmail.com'
[#] Error trying to retrieve this image: mailto:shwetaimages@gmail.com
    From: mailto:shwetaimages@gmail.com
[#] Unhandled Exception: 'int' object has no attribute 'iter_content' (type: AttributeError)
[+] Please delete all the `<BOOK NAME>/OEBPS/*.xhtml` files and restart the program.
[!] Aborting...
``` 

raised due to calling `.iter_content` on `0` returned by `self.requests_provider` for invalid responses/images. Occurs when with book ID 9781449314668 and 9781783288007, for example.